tensorflow_model_optimization/python/core/clustering/keras/cluster.py

```diff
@@ -108,17 +108,8 @@ def cluster_weights(to_cluster,
       number_of_clusters: the number of cluster centroids to form when
         clustering a layer/model. For example, if number_of_clusters=8 then only
         8 unique values will be used in each weight array.
-      cluster_centroids_init: enum value that determines how the cluster
-        centroids will be initialized.
-        Can have following values:
-          1. RANDOM : centroids are sampled using the uniform distribution
-          between the minimum and maximum weight values in a given layer
-          2. DENSITY_BASED : density-based sampling. First, cumulative
-          distribution function is built for weights, then y-axis is evenly
-          spaced into number_of_clusters regions. After this the corresponding x
-          values are obtained and used to initialize clusters centroids.
-          3. LINEAR : cluster centroids are evenly spaced between the minimum
-          and maximum values of a given weight
+      cluster_centroids_init: `cluster_config.CentroidInitialization` instance
```

Comment:
Nit: people using the public API won't be aware of the `cluster_config` module since
they'll just export it as tfmot.clustering.keras.CentroidInitialization. 

Will update it to that when merging so that there is a pointer to the CentroidInitialization api docs
from here.